src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -172,11 +336,11 @@ export default class FileField extends React.Component {
                     </div>
                   )}
                   {!file.uploading && <p>{uiOptions.itemDescription}</p>}
-                  {!file.uploading && (
-                    <span>
-                      <strong>{file.name}</strong>
-                    </span>
+                  {!file.uploading && <strong>{file.name}</strong>}
```

Comment:
I was trying to figure out why the `file.name` wasn't showing up after an attempted upload... because the `action` wasn't including it... and during troubleshooting, I removed the unnecessary span.